lib/matplotlib/bezier.py

```diff
@@ -316,7 +317,7 @@ def _f(xy):
 
 def get_cos_sin(x0, y0, x1, y1):
     dx, dy = x1 - x0, y1 - y0
-    d = (dx * dx + dy * dy) ** .5
+    d = (dx * dx + dy * dy) ** .5 or 1  # Account for divide by zero
```

Comment:
Thanks, I'll make the changes now. The reason for this implementation is that this fix was already present in  `class Arrow(Patch)`: 
```python
Patch.__init__(self, **kwargs)
L = np.sqrt(dx ** 2 + dy ** 2) or 1 # account for div by zero
cx = float(dx) / L
sx = float(dy) / L
```